src/game.h

```diff
@@ -341,8 +342,20 @@ class game
                                       std::vector<Creature *> t, int target,
                                       item *relevant, target_mode mode );
 
-        /** Prompts for target and returns trajectory to it */
-        std::vector<tripoint> pl_target_ui( target_mode mode, item *relevant, int range );
+        /** Targetting UI callback, should return pointer to effective ammo data */
+        using target_callback = std::function<const itype *(item *obj)>;
```

Comment:
> What is the argument?

The item which is being targeted. Not strictly needed here as the implemented callbacks use lambda capture. Could either drop or more preferably document?



> Why not references?

No good reason, should be, will update